Frontend/ui-common/src/Engineering/Helpers/Utils.purs

```diff
@@ -56,4 +68,22 @@ showAndHideLoader delayInMs title description state = do
     _ <- delay $ Milliseconds delayInMs
     _ <- toggleLoader false
     pure unit
-  pure unit
\ No newline at end of file
+  pure unit
+
+getAppConfig :: FlowBT String AppConfig
+getAppConfig = liftFlowBT $ getAppConfig_
+
+getAppConfig_ :: Effect AppConfig
+getAppConfig_  = do
+  config' <- getConfig
+  pure $
+    case config' of
+      Just config -> do
+        case runExcept (decode (encode config )) of
+            Right (obj :: AppConfig) -> config
+            Left err ->DefaultConfig.config
+      Nothing -> do
+            DefaultConfig.config
+
+getConfig :: forall  a. Effect (Maybe a)
+getConfig = getMerchantConfig Just Nothing
```

Comment:
can we not pass `Just` and `Nothing` to JS to get the response in `Maybe` type instead we can have two function one which gets response and one which will add wrapper based on response